src/nspawn/nspawn-settings.c

```diff
@@ -470,6 +470,69 @@ int config_parse_veth_extra(
         return 0;
 }
 
+int config_parse_network_iface_pair(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        char*** l = data;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        return interface_pair_parse_and_test(l, rvalue);
```

Comment:
i am not a fan of this. we generally prefer if log_syntax() is used for parse failures on config files, since it tells line numbers and such.

I am also a bit unhappy that we check system state from a config file parser. Parsers should parse, and not check system state. This should be done in a later step. 

Previously we didn't do such checks here and I doubt we should start now.

I think I would prefer if test_network_interface_initialized() would not be done when parsing but be called for all configured network interfaces in a separate step when are are actually about to actually start the container, after the parsing step. maybe in a new function, called from run(), right after we call verify_arguments().

Reply:
Sorry, I missed this last comment -  but still I understand the point.

Reply:
Ok, Implemented